common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -5,6 +5,7 @@
 # pylint: disable=missing-docstring
 # pylint: disable=invalid-name
 
+import requests
```

Comment:
Please add import statements by keeping in mind the alphabetical order.
Thanks.